x-pack/plugins/cloud_security_posture/server/fleet_integration/fleet_integration.ts

```diff
@@ -26,6 +26,14 @@ type ArrayElement<ArrayType extends readonly unknown[]> = ArrayType extends Read
   ? ElementType
   : never;
 
+const getActivatedBenchmark = (packagePolicy: PackagePolicy): string => {
```

Comment:
should be update, instead fetching `packagePolicy.inputs[0].streams[0].vars?.benchmark.value` we will need to fetch @ofiriro new input type